easybuild/easyconfigs/s/scNanoGPS/scNanoGPS-1.1-foss-2023a.eb

```
@@ -0,0 +1,76 @@
+easyblock = 'Tarball'
+
+name = 'scNanoGPS'
+version = '1.1'
+
+homepage = 'https://github.com/gaolabtools/scNanoGPS/'
+description = "Single cell Nanopore sequencing data for Genotype and Phenotype."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/gaolabtools/scNanoGPS/archive/refs/tags/']
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['60cce1c771283b6d7a3639aa5744acfd6162306d95dd419ad3c265461fdda218']
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('SciPy-bundle', '2023.07'),
+    ('Biopython', '1.83'),
+    ('matplotlib', '3.7.2'),
+    ('Pysam', '0.22.0'),
+    ('Seaborn', '0.13.2'),
+    ('Perl', '5.36.1'),
+    ('minimap2', '2.26'),
+    ('SAMtools', '1.18'),
+    ('tabix', '0.2.6'),
+    ('spoa', '4.1.0'),
+    ('Subread', '2.0.6'),
+    ('Longshot', '1.0.0'),
+    ('BCFtools', '1.18'),
+    ('gffread', '0.12.7'),
+    ('LIQA', '1.3.4'),
+    # Optional - load 'annovar/20200607-GCCcore-12.3.0-Perl-5.36.1' to make 'reporter_SNV.py' work
```

let's just add `annovar` as a dependency here
```suggestion
    ('annovar', '20200607'),
```

If this is a blocker for people who can't obtain the sources for `annovar`, they can always use `eb --filter-deps annovar` as an escape hatch...